scripts/preprocess_data.py

```diff
@@ -87,7 +87,7 @@ def main():
 
     # generate background frequency
     print("generating background frequency...")
-    bgfreq = dict(zip(count_vectorizer.get_feature_names(), master.toarray().sum(1) / args.vocab_size))
+    bgfreq = dict(zip(count_vectorizer.get_feature_names(), [x[0] for x in np.array(master.sum(1)) / args.vocab_size]))
```

Comment:
What I'm just now realizing is that `bgfreq` here is also taking into account frequencies in the dev data.`master` is the result of stacking train on dev and the frequencies are a result of summing in the first dimension to compute word counts is my understanding. Shouldn't bgfreq be built only from the training data?